rust/src/jsonbuilder.rs

```diff
@@ -474,6 +474,34 @@ impl JsonBuilder {
         Ok(self)
     }
 
+    /// Set a key and a string field as the hex encoded string of the value.
+    pub fn set_hex(&mut self, key: &str, val: &[u8]) -> Result<&mut Self, JsonError> {
+        static CHARS: &'static [u8] = b"0123456789abcdef";
```

Comment:
`HEX` exists as a global which provides the same array.

Reply:
thanks